doc/help/commands.asciidoc

```diff
@@ -892,13 +894,21 @@ How many tabs to switch forward.
 
 [[tab-only]]
 === tab-only
-Syntax: +:tab-only [*--prev*] [*--next*]+
+Syntax: +:tab-only [*--prev*] [*--next*] [*--force*]+
 
 Close all tabs except for the current one.
 
 ==== optional arguments
 * +*-p*+, +*--prev*+: Keep tabs before the current.
 * +*-n*+, +*--next*+: Keep tabs after the current.
+* +*-f*+, +*--force*+: Avoid confirmation for pinned tabs.
+
+[[tab-pin]]
+=== tab-pin
+Pin/Unpin the current/[count]th tab.
```

Comment:
I found this in the old docs useful:

> * Pinning a tab shrinks it to `tabs->pinned-width` size.
> * Attempting to close a pinned tab will cause a confirmation, unless `--force` is passed.

What about adding it to the `:tab-pin` docstring so it shows up here?